docs/notebooks/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Notebook
+nav_order: 38
+has_children: false
+---
+
+# Kibana Notebook
+
+A Kibana notebook is an interface that lets you easily create and share live log data reports.
+You can add visualization paragraphs for creating graphs and markdown paragraphs for writing descriptions.
+
+Common use cases include creating postmortem reports, designing runbooks, building live infrastructure reports, and writing documentation.
+
+![Notebook UI](../images/kibana-notebooks.gif)
+
+## Get Started with Notebooks
+
+To get started, choose **Kibana Notebooks** in Kibana.
+
+### Step 1: Create a notebook
+
+A notebook is an interface for creating reports.
+
+1. Choose **Create notebook**.
+- In the pop-up, enter a descriptive name.
+1. Choose **Create**.
+
+Choose **Notebook actions** to rename, duplicate, or delete a notebook.
+
+### Step 2: Add a paragraph
+
+Paragraphs combine text and visualizations for describing data.
+
+#### Add a markdown paragraph
```

Comment:
Capitalize Markdown and add a link to documentation for whatever flavor we're using (probably CommonMark or GFM) in the description.

Examples:

https://github.github.com/gfm/
https://commonmark.org